snapshot/containerd/content.go

```diff
@@ -80,3 +96,58 @@ func (w *nsWriter) Commit(ctx context.Context, size int64, expected digest.Diges
 	ctx = namespaces.WithNamespace(ctx, w.ns)
 	return w.Writer.Commit(ctx, size, expected, opts...)
 }
+
+type nsFallbackStore struct {
+	main *Store
+	fb   *Store
+}
```

Comment:
Out of curiosity, could we potentially use an implementation that was more similar to the content store multiplexing we already have in buildkit (added in #615)?

On the client: https://github.com/moby/buildkit/blob/8dd5f87d1065d15877b4dc739bb1fee7a035fafb/session/content/attachable.go#L21-L41
On the server: https://github.com/moby/buildkit/blob/8dd5f87d1065d15877b4dc739bb1fee7a035fafb/session/content/caller.go#L16-L31

I think we need to still need to have the fallback store as the default, to emulate v0.11 buildkit behavior, but I think maybe moving forwards the client should be able to dictate which store to use?